tests/test_merge.py

```diff
@@ -188,7 +188,9 @@ def test_merge_destination_1(tmp_path):
             ):
                 chunk_bounds = windows.bounds(chunk_window, dst.transform)
                 chunk_arr, chunk_transform = merge([src], bounds=chunk_bounds)
-                target_window = windows.from_bounds(*chunk_bounds, dst.transform)
+                target_window = windows.from_bounds(*chunk_bounds, dst.transform).round(
+                    3
+                )
```

Comment:
GDAL permits floating point windows, but naive use of them is an easy way to get seams or missing data when compositing. Rounding windows to a small fraction of a pixel solves these problems in most cases.